src/components/media-views.js

```diff
@@ -563,10 +565,14 @@ AFRAME.registerComponent("media-video", {
       this.el.setObject3D("mesh", this.mesh);
     }
 
-    this.mesh.material.map = texture;
+    if (this.data.contentType === "audio/mpeg") {
```

Comment:
Any reason not to just look for anything starting with `audio/`, the browser can probably handle a number of different formats.

Reply:
I kinda like only supporting media that all browsers can load. It'll make it easier to track down bugs. However, I also like being able to support better formats in most browsers 🤷‍♂

Reply:
:+1: Changed it to allow all `audio/` mime types. This accounts for wav and flac, but not for ogg, since ogg uses `application/ogg`. I'm going to punt on ogg for now.